config.py

```diff
@@ -17,3 +17,6 @@ class Config:
     DB_USESSL = os.getenv('DB_USESSL', '')
     CASEAPI_HOST = os.getenv('CASEAPI_HOST', 'localhost')
     CASEAPI_PORT = os.getenv('CASEAPI_PORT', '8161')
+    EXCEPTIONMANAGER_HOST = os.getenv('EXCEPTIONMANAGER_HOST', 'localhost')
+    EXCEPTIONMANAGER_PORT = os.getenv('EXCEPTIONMANAGER_PORT', '8666')
```

Comment:
The environment variables in k8s are called:
EXCEPTION_MANAGER_SERVICE_HOST
EXCEPTION_MANAGER_SERVICE_PORT